wigner_functions.cc compilation error

during compilation time, the follow error occured:

collect2: error: ld returned 1 exit status
gmake[2]: *** [src/CMakeFiles/arts.dir/build.make:120: src/arts] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1570: src/CMakeFiles/arts.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
/root/miniconda3/envs/pyarts-dev/bin/../lib/gcc/x86_64-conda-linux-gnu/13.1.0/../../../../x86_64-conda-linux-gnu/bin/ld: ../libartscore.a(wigner_functions.cc.o): in function `make_wigner_ready(int, int, int)':
wigner_functions.cc:(.text+0x12c): undefined reference to `fastwigxj_thread_dyn_init'
/root/miniconda3/envs/pyarts-dev/bin/../lib/gcc/x86_64-conda-linux-gnu/13.1.0/../../../../x86_64-conda-linux-gnu/bin/ld: wigner_functions.cc:(.text+0x17f): undefined reference to `fastwigxj_thread_dyn_init'
/root/miniconda3/envs/pyarts-dev/bin/../lib/gcc/x86_64-conda-linux-gnu/13.1.0/../../../../x86_64-conda-linux-gnu/bin/ld: wigner_functions.cc:(.text+0x18c): undefined reference to `fastwigxj_thread_dyn_init'

--- a/3rdparty/wigner/fastwigxj/CMakeLists.txt
+++ b/3rdparty/wigner/fastwigxj/CMakeLists.txt
@@ -12,6 +12,16 @@ check_c_source_compiles("
 )
 
 check_c_source_compiles("
+    __thread int global = 0;
+    int main() {
+        global = 2;
+        return 0;
+    }
+    "
+    FASTWIGXJ_HAVE_THREAD
+)
+
+check_c_source_compiles("
     #include <x86intrin.h>
     typedef long long v4di __attribute__ ((vector_size (32)));
     int main() {

--- a/3rdparty/wigner/fastwigxj/fastwigxj_auto_config.h.cmake
+++ b/3rdparty/wigner/fastwigxj/fastwigxj_auto_config.h.cmake
@@ -1,5 +1,6 @@
 // auto-generated definitions file
 
+#cmakedefine FASTWIGXJ_HAVE_THREAD 1
 #cmakedefine FASTWIGXJ_HAVE_THREAD_MUTEX 1
 #cmakedefine FASTWIGXJ_HAVE_LSFENCE 1
 #cmakedefine FASTWIGXJ_HAVE_SSE4_1 1

